client/lib/users/actions.js

```diff
@@ -113,52 +113,78 @@ const UsersActions = {
 		wpcom.undocumented().site( siteId ).updateUser( userId, attributes, ( error, data ) => {
 			if ( error ) {
 				debug( 'Update user error', error );
-				Dispatcher.handleServerAction( {
+				return Dispatcher.handleServerAction( {
 					type: 'RECEIVE_UPDATE_SITE_USER_FAILURE',
 					action: 'UPDATE_SITE_USER',
 					siteId: siteId,
 					user: user,
 					error: error
 				} );
-			} else {
-				Dispatcher.handleServerAction( {
-					type: 'RECEIVE_UPDATE_SITE_USER_SUCCESS',
-					action: 'UPDATE_SITE_USER',
-					siteId: siteId,
-					user: user,
-					data: data
-				} );
 			}
+
+			Dispatcher.handleServerAction( {
+				type: 'RECEIVE_UPDATE_SITE_USER_SUCCESS',
+				action: 'UPDATE_SITE_USER',
+				siteId: siteId,
+				user: user,
+				data: data
```

Comment:
fun fact! we can use object shorthand notation here to prevent repeating ourselves

```js
{
	type: '…',
	action: '…',
	siteId,
	user,
	data,
}
```